app/scripts/views/mixins/signed-out-notification-mixin.js

```diff
@@ -34,11 +33,9 @@ define(function (require, exports, module) {
     },
 
     navigateToSignIn () {
-      this.navigate('signin', {
-        success: BaseView.t('Signed out successfully')
-      }, {
-        clearQueryParams: true
-      });
+      // Navigate via the back-end in order to regenerate
+      // flow id and flow begin time for the next session.
+      this.window.location.href = '/signin';
```

Comment:
> First, there should probably be an abstraction for the action of navigating via location.href instead via pushState. The reason is we want to ensure all metrics are flushed before redirecting and not all environments support the beforeunload handler...

Good point, I'll add something.

> ...we should probably propagate the current query parameters.

I opted not to do that because the existing code specifies `clearQueryParams: true`. Do you mean for the general case, and then override that in my usage, or do you mean my usage itself should propagate query params?